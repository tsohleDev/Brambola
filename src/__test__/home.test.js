import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import Home from '../components/home/home';

describe('Home Test', () => {
  test('display pairs', () => {
    const home = render(
      <Provider store={store}>
        <BrowserRouter>
          <Home />
        </BrowserRouter>
      </Provider>,
    );
    expect(home).toMatchSnapshot();
  });
});
