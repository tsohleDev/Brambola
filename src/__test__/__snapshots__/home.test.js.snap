// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Test display pairs 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <button
          type="button"
        >
          <span
            class="material-symbols-outlined"
          >
            arrow_back_ios_new
          </span>
        </button>
        <h2>
          Stocks
        </h2>
        <div>
          <span
            class="material-symbols-outlined"
          >
            mic
          </span>
          <span
            class="material-symbols-outlined"
          >
            settings
          </span>
        </div>
      </header>
      <main>
        <section
          class="heading"
        >
          <img
            alt="USD"
            src="USD.svg"
          />
          <div>
            <h1>
              USD FOREIGN EXCHANGE PRICES
            </h1>
            <pre>
              6
               
              pairs
            </pre>
          </div>
        </section>
        <h3>
          PAIRS WITH USD AS BASE CURRENCY
        </h3>
        <section
          class="home-page"
        />
      </main>
    </div>
  </body>,
  "container": <div>
    <header>
      <button
        type="button"
      >
        <span
          class="material-symbols-outlined"
        >
          arrow_back_ios_new
        </span>
      </button>
      <h2>
        Stocks
      </h2>
      <div>
        <span
          class="material-symbols-outlined"
        >
          mic
        </span>
        <span
          class="material-symbols-outlined"
        >
          settings
        </span>
      </div>
    </header>
    <main>
      <section
        class="heading"
      >
        <img
          alt="USD"
          src="USD.svg"
        />
        <div>
          <h1>
            USD FOREIGN EXCHANGE PRICES
          </h1>
          <pre>
            6
             
            pairs
          </pre>
        </div>
      </section>
      <h3>
        PAIRS WITH USD AS BASE CURRENCY
      </h3>
      <section
        class="home-page"
      />
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
